packages/components/src/AzurePersistentDiskSizeSelectInput.ts

```diff
@@ -0,0 +1,12 @@
+import { SingleValue } from 'react-select';
+
```

Comment:
Let me know if this file needs to move in some sub-directory rather than directly in components/src

Reply:
These disk related input files should _not_ move into the components package. They are specific to the Terra UI application.